shared-helpers/src/utilities/MessageContext.ts

```diff
@@ -0,0 +1,59 @@
+import React, { FunctionComponent, createElement, useRef, useState } from "react"
+
+export enum alertTypes {
```

Comment:
This should be typed according to what's available in seeds, it looks like that actually isn't available as an exported type though - just some cleanup for later that's on my radar.